### ISSUE TITLE
Next Build Reverse Proxy Empty bucket testing for dev

### ISSUE THREAD

**Issue by mreed101** [department-of-veterans-affairs/next-build#740]
## Description

The goal of this ticket is to properly test changes to the reverse proxy via our test plan.
When testing a thread should be started in the #accelerated-publishing slack channel to keep the team updated with passing or failing test. All findings should be recorded with this ticket.

## Acceptance Criteria

- [x] Results of each test added to this ticket
- [x] Team informed of passing or failing test

**Comment by mreed101** [comment]
Update around 403 testing

Originally the testing around 403 required that we test https://www.va.gov/va_404.html to return as a 403 forbidden page via openresty.
During testing we discovered that va_404.html is not a real page. It is only being created via the reverse proxy and is forcing an error via an internal directive.
The route does not appear to be used anywhere inside of our VA repositories https://github.com/search?q=org%3Adepartment-of-veterans-affairs%20va_404&type=code
When hitting our s3 bucket it will always return a 403 but there is logic inside of the reverse proxy that converts this 403 into a 404 error.
We could modify our bucket policy to return 404s but that is not the configuration that we would use in production so we have opted not to make this change.
In conclusion there is not a proper way for us to test that a 403 error has been returned as 403s from our s3 bucket will be converted into 404s.
